### PR TITLE
RSpec: split up rmagick/draw_spec.rb

### PR DESCRIPTION
**Pull request by mockdeep**
This is the first of many commits that will break up specs into one file
per method. It's a little unconventional to structure specs in this way
in Ruby, but here are a few benefits:

- Spec files are smaller, which makes them easier to navigate when
  looking for details relevant to what you're trying to test. This is
  especially valuable for classes that have dozens of methods where the
  test files can grow to several thousands of lines long.
- Easier to find specs for the method you need to test. You can
  now use your fuzzy finder to locate more specifically where the tests
  are.
- Encourages thinking about tests in a structured, per-method way.
  Structuring spec files this way is constraining in a way that pushes
  developers to think about which method is most relevant to what
  they're testing. It also saves some mental energy as developers don't
  have to spend as much time considering where tests belong in a much
  larger spec file, or what the description should be.

### CODE REVIEW

File: spec/rmagick/draw/class_methods/_dummy_img__spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+RSpec.describe Magick::Draw, '._dummy_img_' do
+  let(:draw) { described_class.new }
+
+  it 'works' do
+    # initially this variable is not defined.
```
Comment (dlemstra) [inline spec/rmagick/draw/class_methods/_dummy_img__spec.rb]: Maybe we could now remove the commented out code?

File: spec/rmagick/draw/interline_spacing_writer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+RSpec.describe Magick::Draw, '#interline_spacing=' do
+  let(:draw) { described_class.new }
+
+  it 'assigns without raising an error' do
+    expect { draw.interline_spacing = 1 }.not_to raise_error
+  end
```
Comment (dlemstra) [inline spec/rmagick/draw/interline_spacing_writer_spec.rb]: Not sure if this is now the moment to do it but maby we could also add a test for an invalid parameter? And also in some other tests?

File: spec/rmagick/draw/class_methods/_dummy_img__spec.rb (hunk 1 above)
Comment (mockdeep) [reply spec/rmagick/draw/class_methods/_dummy_img__spec.rb]: Sure, cleaned up.

File: spec/rmagick/draw/interline_spacing_writer_spec.rb (hunk 2 above)
Comment (mockdeep) [reply spec/rmagick/draw/interline_spacing_writer_spec.rb]: @dlemstra I think I'd like to leave off adding new specs to the next phase of the process. That way we can make sure we first get through splitting them up in a reasonable amount of time. After this, the next step will be going through all of the specs with with a fine toothed comb, cleaning them up and adding new ones.

**Pull request merged**